docs/pages/versions/unversioned/sdk/filesystem.md

```diff
@@ -595,24 +595,40 @@ FileSystem.getTotalDiskCapacityAsync().then(totalDiskCapacity => {
 
 Returns a Promise that resolves to a number that specifies the total internal disk storage capacity in bytes, or JavaScript's [`MAX_SAFE_INTEGER`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) if the capacity is greater than 2<sup>53</sup> - 1 bytes.
 
+### `FileSystem.askForDirectoryPermissionsAsync(initialFileUrl)`
+
+**Android only**. Allows users to select a specific directory, granting your app access to all of the files and sub-directories within that directory.
+
+#### Arguments
+
+- **initialFileUrl (_string_)** -- **Optional**. The [SAF URI](#saf-uri) of the directory that the file picker should display when it first loads. **Available only on Android R or higher**.
```

Comment:
how does a user determine the initialFileUrl? where does that come from?

Reply:
I've added a function that will generate a SAF URI from the album name.